### DISCUSSION TITLE
Live search over table?

### DISCUSSION THREAD

**Discussion by Pulsdesign** [Budibase/budibase#7032] [Help] [answered]
Hi there,
Is there a possibility to search a table with live results. Like typing in 3 letters and all matches will appear and not to have press enter or type the exact search-term in?

Thx for helping out :)

**Comment by melohagan** [comment, marked as answer]
Hey @Pulsdesign 

This is not currently possible, but it's a cool idea! I've raised a feature request here: https://github.com/Budibase/budibase/issues/7092
